tools/train.py

```diff
@@ -132,7 +132,8 @@ def train_and_eval(runid, module_prep_model, task, c, do_eval=True):
     print('Dataset')
     if 'vocabf' in conf:
         task.load_vocab(conf['vocabf'])
-    task.load_data(trainf, valf)
+    if 'testf' in conf:
+        task.load_data(trainf, valf,tesf)
```

Comment:
Hmm, a bit confused by this.  What's the purpose of this?  It even has a typo, so probably this change never actually comes into effect?